plugins/woocommerce/src/Admin/API/Reports/Products/DataStore.php

```diff
@@ -111,6 +111,40 @@ protected function assign_report_columns() {
 	 */
 	public static function init() {
 		add_action( 'woocommerce_analytics_delete_order_stats', array( __CLASS__, 'sync_on_order_delete' ), 10 );
+		add_action( 'woocommerce_order_partially_refunded', array( __CLASS__, 'add_partial_refund_type_meta' ), 10, 2 );
+		add_action( 'woocommerce_order_fully_refunded', array( __CLASS__, 'add_full_refund_type_meta' ), 10, 2 );
```

Comment:
Thanks for the reminder, I've added in 5e1bde3. Since we only care about the full refunds without line items, I added the `_refund_type` meta to orders without line items found in the query from the migration script.